CIP-1775/README.md

```diff
@@ -0,0 +1,108 @@
+
+### Use Cases
+
+Instead of sending tokens to the dapp smart-contract, the user creates an output in their own wallet that delegates spending authorization to a smart-contract (or other entity).
+
+Some **existing** use cases, where this can be beneficial:
+
+1. Swapping tokens on a decentralized exchange.
+2. Providing liquidity on a decentralized exchange.
+3. Listing an NFT on a marketplace.
+
+Some **New** use cases, enabled by this proposal:
+
+1. On-chain noncustodial asset management. A user can delegate managing (some of) their assets to a manager, another individual user or organization.
+2. Concurrent use of dapps. For example, a user can provide liquidity and, at the same time, have a trade open at a (different) DEX, with the same output. When the trade is executed, it looks just as though the user has withdrawn the output from the liquidity pool.
+3. Seamless interaction with L2 (sidechains or Hydra). The user doesn't need to send assets to the L2 entrypoint script/multisig, nor do they need to explicitly take them out.
+4. Combinations of all of the above, both existing and new 🙃.
+
+### Other Benefits
+
+- Users' assets continue to participate in delegated proof of stake and on-chain governance.
+- Assets cannot be locked forever in noncustodial (delegated) smart-contracts, by definition.
+
+## Specification
+
+### Delegated Spending Authorizations
+
+A solution is to introduce _delegated spending authorizations_: instead of **sending** a output to an address, a user could **delegate** a spending authorization of the output to the address.
+
+### CDDL
+
+To enable this, a **new** field is added to the transaction output (the current spec can be found here: https://github.com/input-output-hk/cardano-ledger/blob/master/eras/babbage/test-suite/cddl-files/babbage.cddl):
+
+```cddl
+post_alonzo_transaction_output =
+  { 0 : address
+  ; ...
+  , ? 4 : { * address => ( datum_option / nil ) }   ; New; delegated spending authorizations
+  }
+```
+
+The new field is a map from `address` to `datum_option` or `nil`. In this map, `address` can be any address: Plutus v1 and v2 scripts, wallets, etc. The value (`datum_option`) only makes sense for Plutus script addresses, so it is optional (can be `nil`).
+
+### Spending of Outputs
+
+When the output's `address` (field 0) is a **wallet** address, a transaction can spend the output if it satisfies spending conditions:
+- for the output's address (i.e. signed by the wallet's private key), or 
+- for any address in the output's _delegated spending authorizations_.
+
+If the output's `address` is of **any other kind** (e.g. native or Plutus script), the _delegated spending authorizations_ field contents **do not affect** spending behavior. Existing scripts' assumptions must be **unaffected** by the proposed ledger change: spending of outputs at a script address are controlled by the script only.
```

Comment:
I don't particularly like that this is asymmetrical. I haven't thought of somewhere where that goes wrong, but there often is such a place.

Reply:
Yes, I realize that. To fix this, I'm updating the text so that only beginning with Plutus v3, scripts will be able to participate in DSAs or receive outputs with DSAs.

Reply:
@michaelpj I've updated the text.